apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/BeneficiaryTransformerV2.java

```diff
@@ -98,87 +98,90 @@ private static Patient transform(Beneficiary beneficiary, RequestHeaders request
 
     Patient patient = new Patient();
 
-    patient.setId(beneficiary.getBeneficiaryId());
-    patient.addIdentifier(
-        TransformerUtilsV2.createIdentifier(
-            CcwCodebookVariable.BENE_ID, beneficiary.getBeneficiaryId()));
+    // Required values not directly mapped
+    patient
+        .getMeta()
+        .addProfile("http://hl7.org/fhir/us/carin-bb/StructureDefinition/C4BB-Patient");
```

Comment:
Add the profile to:
https://github.com/CMSgov/beneficiary-fhir-data/blob/master/apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/commons/ProfileConstants.java

Reply:
Done; thx, I should have just done that since you've been taking the time to do things right.